app/assets/v2/js/explorer.js

```diff
@@ -0,0 +1,276 @@
+let bounties = [];
+let featuredBounties = [];
+let bountiesPage = 1;
+let BountiesLimit = 10;
+let BountiesOffset = 0;
+let bountiesNumPages = '';
+let bountiesHasNext = false;
+let numBounties = '';
+// let funderBounties = [];
+let paramsDefault = {
+  order_by: '-web3_created',
+  idx_status: '',
+  experience_level: [],
+  project_type: [],
+  applicants: '',
+  bounty_filter: '',
+  bounty_categories: [],
+  moderation_filter: '',
+  permission_type: [],
+  project_length: [],
+  bounty_type: [],
+  network: ['mainnet'],
+  keywords: []
+};
+
+Vue.filter('stringFixer', (str) => {
+  return str.replace(/_/i, ' ');
+});
+
+Vue.mixin({
+  methods: {
+    fetchBounties: function(newPage, featured) {
+      let vm = this;
+
+      vm.isLoading = true;
+      vm.noResults = false;
+      vm.params.limit = vm.BountiesLimit;
+      if (newPage || newPage === 0) {
+        vm.BountiesOffset = newPage;
+      }
+      vm.params.offset = vm.BountiesOffset;
+      if (vm.searchTerm) {
+        vm.params.search = vm.searchTerm;
+      } else {
+        delete vm.params['search'];
+      }
+
+      let searchParams = new URLSearchParams(vm.params);
+
+      window.history.replaceState({}, '', `${location.pathname}?${searchParams}`);
+
+      let apiUrlBounties = `/api/v0.1/bounties/slim/?${searchParams.toString()}`;
+
+      if (featured) {
+        apiUrlBounties = `/api/v0.1/bounties/slim/?${searchParams.toString()}&is_featured=True`;
+      }
+
+      var getBounties = fetchData (apiUrlBounties, 'GET');
+
+      $.when(getBounties).then(function(response) {
+
+        response.forEach(function(item) {
+          if (featured) {
+            vm.featuredBounties.push(item);
+          } else {
+            vm.bounties.push(item);
+          }
+        });
+
+        if (featured) {
+          return vm.featuredBounties;
+        }
+        vm.numBounties = response.length;
+        if (vm.numBounties < vm.BountiesLimit) {
+          vm.bountiesHasNext = false;
+        } else {
+          vm.bountiesHasNext = true;
+        }
+        if (vm.bountiesHasNext) {
+          vm.BountiesOffset = vm.BountiesOffset + vm.BountiesLimit;
+
```

Comment:
The style here in regards to spacing could be cleaned up - unless this has some implied meaning in Vue I'm unaware of